File: Python/Program/Vehicle.py
class Vehicle:
    def __init__(self, plat_nomor, merk, tahun_produksi, warna):
        self.plat_nomor = plat_nomor
        self.merk = merk
        self.tahun_produksi = tahun_produksi
        self.warna = warna
    
    def get_plat_nomor(self):
        return self.plat_nomor

    def set_plat_nomor(self, plat_nomor):
        self.plat_nomor = plat_nomor

    def get_merk(self):
        return self.merk

    def set_merk(self, merk):
        self.merk = merk

    def get_tahun_produksi(self):
        return self.tahun_produksi

    def set_tahun_produksi(self, tahun_produksi):
        self.tahun_produksi = tahun_produksi

    def get_warna(self):
        return self.warna

    def set_warna(self, warna):
        self.warna = warna

File: CPP/Program/Vehicle.cpp
// Menggunakan pragma once agar header file hanya di-include satu kali
#pragma once
// Memasukan library yang digunakan
#include <iostream>
#include <string>

// Deklarasi namespace
using namespace std;

class Vehicle {
private:
    string plat_nomor;
    string merk;
    int tahun_produksi;
    string warna;
public:

    Vehicle(string plat_nomor, string merk, int tahun_produksi, string warna) : plat_nomor(plat_nomor), merk(merk), tahun_produksi(tahun_produksi), warna(warna) {

    }
    virtual ~Vehicle() {}

    string getPlatNomor()  {
        return plat_nomor;
    }

    void setPlatNomor(string plat_nomor) {
        this->plat_nomor = plat_nomor;
    }

    string getMerk() {
        return merk;
    }

    void setMerk(string merk) {
        this->merk = merk;
    }

    int getTahunProduksi() {
        return tahun_produksi;
    }

    void setTahunProduksi(int tahun_produksi) {
        this->tahun_produksi = tahun_produksi;
    }

    string getWarna() {
        return warna;
    }

    void setWarna(string warna) {
        this->warna = warna;
    }
    
    // Destructor
    // ~Vehicle()
    // {


    // }
};